Packs/VirusTotal/Integrations/VirusTotal_V3_Premium/VirusTotal_V3_Premium.yml

```diff
@@ -3,7 +3,7 @@ commonfields:
   id: VirusTotal - Premium (API v3)
   version: -1
 configuration:
-- display: ""
+- display: API Key (leave empty. Fill in the API key in the password field.)
```

Comment:
There is no changes in the UI, maybe because the **displaypassword** key

Reply:
I see now [here](https://xsoar.pan.dev/docs/integrations/code-conventions#credentials ) that we can remove the **display** key. it's unnecessary in case there is a **displaypassword** key.